app/main.py

```diff
@@ -1,6 +1,20 @@
-from typing import Callable
+from typing import Callable, Any
+import functools
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_store = {}
+
+    @functools.wraps(func)
+    def inner(*args, **kwargs) -> Any:
+        key = args + tuple(kwargs.values()) + (func.__name__,)
```

Comment:
```suggestion
        key = args + tuple(kwargs.values())
```
For each function cache_store will be unique

Reply:
Yes I think so. 
Approve please my new solut.
Thanks!